### PR TITLE
 add testNamesOfNavigationBar

### PR DESCRIPTION
**Pull request by IrynaKolyada**
add testNamesOfNavigationBar

### CODE REVIEW

**Review by AllaBird — APPROVED** [review]
LGFM

File: src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java [hunk 1]
```diff
@@ -531,4 +532,34 @@ public void testAuthorizationSaucedemoLockedUser() {
         Assert.assertEquals(actualErrorText, expectedErrorText);
 
     }
+
+    @Test
+    public void testNamesOfNavigationBar() throws InterruptedException {
+        final String url = "https://openweathermap.org/";
```
Comment (Gr0zpu) [inline src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java:538]: for final var need use UPPE_RCASE

File: src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java [hunk 2]
```diff
@@ -531,4 +532,34 @@ public void testAuthorizationSaucedemoLockedUser() {
...
+        final List<String> expectedResult = List.of("Guide", "API", "Dashboard", "Marketplace", "Pricing",
```
Comment (Gr0zpu) [inline src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java:539]: for final var need use UPPE_RCASE

**Review by Elgrueber — APPROVED** [review]
LGTM

**Review by Gr0zpu — CHANGES_REQUESTED** [review]

File: src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java (hunk 1 above)
Comment (IrynaKolyada) [reply src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java:538]: It' s final just for the particular test. Thanks

File: src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java (hunk 2 above)
Comment (IrynaKolyada) [reply src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java:539]: Pls remove the block. it's against the rules

**Review by Gr0zpu — APPROVED** [review]

**Pull request merged**